drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/reports/index.haml

```diff
@@ -44,16 +44,18 @@
                   - else
                     = render 'admin/inactive_users/user_status', user: user
                 %br
-                - if user.last_sign_in_at.present?
-                  Last login: #{user.last_sign_in_at}
+                - if user.current_sign_in_at.present?
+                  Last login: #{user.current_sign_in_at}
                 - else
                   Never logged in
             %td
               = link_to user_permission_report_warehouse_reports_user_path(user), data: {loads_in_pjax_modal: true}, class: 'btn btn-xs btn-primary btn-muted btn-hmis' do
                 View Details
             %td
               - if user.using_acls?
-                TODO
+                %ul
+                  - user.legacy_roles.map{|r| link_to_if can_edit_roles?, r.name, edit_admin_role_path(r)}.uniq.each do |role_name|
```

Comment:
This seems slightly odd to me that we'd check for using_acls and then use `legacy_roles`.  Should we adjust that?